src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -333,4 +333,8 @@ export default class FiatConnectQuote extends NormalizedQuote {
       this.flow === CICOFlow.CashIn ? this.getCryptoAmount() : this.getFiatAmount()
     )
   }
+
+  getTokenId(): string | null {
```

Comment:
In this case (as opposed to in `ExternalQuote`) it returns string or null because there is a case where normalizeQuotes is called in a place where `tokenId` is not available, so the normalized fiat connect quote has no `tokenId`.